### DISCUSSION TITLE
3d implementation of object

### DISCUSSION THREAD

**Discussion by sl2676** [rnmapbox/maps#2963] [General]
I've been working on a project where I would want to show a 3d object for the user and I'm not sure how to go about it. I would appreciate any help over importing 3d objects into the map

**Comment by Gnative** [comment]
@sl2676 there is currently no implementation through this package for 3d models but you could go the route to create your own custom layers in ios/android to support the use of 3D models.. 

There is an example on Mapbox iOS on how this is done on the iOS side. 

https://github.com/mapbox/mapbox-maps-ios/blob/main/Apps/Examples/Examples/All%20Examples/SceneKitExample.swift
